src/databricks/labs/ucx/install.py

```diff
@@ -248,21 +248,29 @@ def _create_jobs(self):
         self._create_readme()
         self._create_debug(remote_wheel)
 
+    @staticmethod
+    def _step_list() -> list[str]:
+        step_list = []
+        [step_list.append(task.workflow) for task in _TASKS.values() if task.workflow not in step_list]
+        return step_list
+
     def _create_readme(self):
         md = [
             "# UCX - The Unity Catalog Migration Assistant",
             f'To troubleshoot, see [debug notebook]({self._notebook_link(f"{self._install_folder}/DEBUG.py")}).\n',
             "Here are the URL and descriptions of jobs that trigger's various stages of migration.",
             "All jobs are defined with necessary cluster configurations and DBR versions.",
         ]
-        for step_name, job_id in self._deployed_steps.items():
+        for step_name in self._step_list():
+            job_id = self._deployed_steps[step_name]
```

Comment:
And what will happen if step is not deployed?

Reply:
Understood, will address asap.